packages/common/src/styles/theme.ts

```diff
@@ -79,10 +79,10 @@ const themeOptions = {
   breakpoints: {
     values: {
       xs: 0,
-      sm: 600,
-      md: 1024,
-      lg: 1440,
-      xl: 1536,
+      sm: 599,
+      md: 1025,
+      lg: 1441,
+      xl: 1537,
```

Comment:
when I use `.down` then the value is exclusive - so `down(1024)` is false and so I've been using `smallScreen` as `1024` and below which was using `down('md')`.. I can see how this  could be confusing so if you have any better ideas 🤷

Reply:
I'm already confused! why is `sm` `599` when `md` is `1025`?
I think having `down('md')` is fine - it is to me anyway. I justified it with thinking that smaller than medium is small ( and xtra small ).. similarly medium is 1025 and up. All nicely hidden behind hooks anyway, so people won't even need to think about it!

Reply:
ah yes i've changed it to 601 ..

😕